doc/dependencies.md

```diff
@@ -36,3 +36,4 @@ Bitcoin Core requires one of the following compilers.
 | [SQLite](../depends/packages/sqlite.mk) (wallet) | [link](https://sqlite.org) | [3.38.5](https://github.com/bitcoin/bitcoin/pull/25378) | [3.7.17](https://github.com/bitcoin/bitcoin/pull/19077) | No |
 | Python (scripts, tests) | [link](https://www.python.org) | N/A | [3.10](https://github.com/bitcoin/bitcoin/pull/30527) | No |
 | [systemtap](../depends/packages/systemtap.mk) ([tracing](tracing.md)) | [link](https://sourceware.org/systemtap/) | [4.8](https://github.com/bitcoin/bitcoin/pull/26945)| N/A | No |
+| [Cap'n Proto](../depends/packages/capnp.mk) | [link](https://capnproto.org) | [0.6.1](https://capnproto.org/install.html) | 0.5.3 | Yes |
```

Comment:
Shouldn't this be "no" for runtime? If this goes into a release, it won't be a runtime dep.

Reply:
Mmh, shouldn't that be flipped once the PR for adding it to the release is actually in?

Reply:
In that case, adding this seems premature, as this document should represent the state of the release binaries.

Reply:
It looks like this should be changed to "no" to be consistent with the rest of the table since qt, sqlite, etc also have "no". I would think of these all as runtime dependences, but IIUC the distinction this is making is that all of these are statically linked but freetype and fontconfig are dynamically linked.

Reply:
Does also seem ok to drop this to avoid implying it is enabled in release binaries. Could add it in #31802

Reply:
Dropped it for now.

Reply:
I made a note to add it.